src/ISolutionExplorerRepository.ts

```diff
@@ -23,9 +23,10 @@ export interface ISolutionExplorerRepository {
    * Get a single diagram from the current solution.
    *
    * @param diagramName The name of the diagram to load.
+   * @param pathspec The pathspec from which to load.
```

Comment:
```suggestion
   * @param pathspec The pathspec from which to load the diagram.
```

I just love full sentences. 😉